### PR TITLE
Change PerfView pipeline to YAML pipeline

### PR DESCRIPTION
**Pull request by mjsabby**

### CODE REVIEW

**Comment by mjsabby** [comment]
@brianrob This takes your pipeline in ADO and makes it YAML-based so it can be testable.

Once this is checked in, you'll have to delete the pipeline you created and just create a new one using this yaml file.

**Comment by mjsabby** [comment]
Here is a link to a successful build using this YAML file: https://dev.azure.com/ms/perfview/_build/results?buildId=69020

**Comment by mjsabby** [comment]
@brianrob Any comments?

File: .ado.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+    steps:
+    - task: DockerInstaller@0
+      displayName: 'Install Docker'
+
+    - bash: |
+       set -x
+   
+       uname -a
+       apt-get install linux-azure-tools-5.0.0-1032
```
Comment (brianrob) [inline .ado.yml]: Can you please remove this test script?

Comment (mjsabby) [reply .ado.yml]: Done

Comment (brianrob) [reply .ado.yml]: Thanks much.

**Pull request merged**